ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -813,6 +824,35 @@ export default {
         },
       },
     },
+    {
+      id: 'E12S Basic Relativity Yellow Hourglass',
+      // Orient where "Yellow" Anger Hourglass spawns
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '9824' }),
+      condition: (data, matches) => data.phase === 'basic',
```

Comment:
Does this npc spawn in other phases, or can `condition` be removed?

Reply:
The npc spawns only in Basic Relativity along with multiple Joy's Hourglass (npcNameId 9822). The Advanced Relativity are all Sorrow's Hourglass with npcNameId 9823. I'll remove the condition here.